.github/workflow-templates/dev-tests/action.yml

```diff
@@ -26,7 +26,7 @@ runs:
       with:
         node-version: 20.10.0
         cache: "pnpm"
-        cache-dependency-path: test/pnpm-lock.yaml
```

Comment:
the lock file is in test folder, right?

Reply:
The lock file has been moved to the root of the repo